ci/ci/build.py

```diff
@@ -467,9 +455,6 @@ def cleanup(self, batch, scope, parents):
 set -x
 date
 
-gcloud -q auth activate-service-account \
-  --key-file=/secrets/registry-push-credentials/credentials.json
-
```

Comment:
Deleting this entirely because `gcloud` will use the hail metadata server to get credentials for CI's identity.